src/app/store/system-info/system-info.effects.ts

```diff
@@ -77,21 +78,6 @@ export class SystemInfoEffects {
     }),
   ));
 
-  subscribeToHa = createEffect(() => this.actions$.pipe(
```

Comment:
Why remove this part?

Reply:
@undsoft  - as I've seen it's not used in the APP

Reply:
I think was used - it created a subscription to HA updates.